mobileapps/views.py

```diff
@@ -1017,6 +1017,9 @@ class OrganizationThemeDetailView(MobileRetrieveUpdateDestroyAPIView):
         * navigation_icon_color
         * completed_course_tint
         * lesson_navigation_color
+
+        **DELETE**
+        Deactivates the theme by default. Set query param `remove` to `true` to actually theme and its images.
```

Comment:
Shouldn't this be: `Set query param remove to true to actually delete theme and its images.`?

Reply:
Of course it should :)
Fixed!